angular-workspace/projects/ni/nimble-angular/src/thirdparty/directives/radio_control_value_accessor.ts

```diff
@@ -165,6 +174,10 @@ export class RadioControlValueAccessor extends BuiltInControlValueAccessor imple
    */
   @Input() value: any;
 
+  // [Nimble]: Can't override default behavior by injection token, because it is not exported. Inlining value of setDisabledStateDefault.
+  private callSetDisabledState = 'always';
```

Comment:
@mollykreis I want to see if you agree this is the right way to handle integrating these changes introduced by the 15.2 version of the file. Seems like the result is that apps won't be able to opt out of a [breaking change in Angular 15](https://angular.io/guide/update-to-version-15#setdisabledstate-is-always-called-when-a-controlvalueaccessor-is-attached) (which, tbh, I did not totally understand). If we really needed to support it, I guess we could define our own token and module config option, [like Angular does](https://github.com/angular/angular/blob/c3e9a2b7cc659ef387d3bba01b6848a9d052a04f/packages/forms/src/form_providers.ts#L46) for the `FormsModule` and `ReactiveFormsModule`. I'm assuming that's not necessary, though.

Reply:
It certainly looks like the most straightforward way to integrate the changes. I'm not a huge fan of the fact that the `nimble-radio` will have a slightly different behavior than other nimble form controls, but I don't see a way to avoid that.

It [also doesn't look like we have very many usages of the radio button in Angular code](https://dev.azure.com/ni/DevCentral/_search?action=contents&text=nimble-radio&type=code&lp=code-Project&filters=ProjectFilters%7BDevCentral%7DRepositoryFilters%7BSkyline%7D&pageSize=25&result=DefaultCollection/DevCentral/Skyline/GBmaster//Web/Workspaces/RoutineManager/src/app/components/routine-events/routine-events.component.html), so it's probably not a big deal if we don't support the legacy mode.

Reply:
Looks like the `SetDisabledStateOption` is a public api, can we use that type here?
```suggestion
  private callSetDisabledState: SetDisabledStateOption = 'always';
```